roles/rhol_crc/tasks/main.yml

```diff
@@ -137,3 +137,37 @@
 - name: Add crc kubeconfig
   when: cifmw_rhol_crc_creds | bool
   ansible.builtin.import_tasks: add_crc_creds.yml
+
+# Note(Lewis): Only needed for CRC => 2.32.0-4.14.8
+- name: Configure DNS in CRC VM
+  delegate_to: crc
+  vars:
+    _crc_ip: "192.168.122.10"
```

Comment:
Can we avoid hard-coding this?
WDYT? https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci/playbooks/multinode-customizations.yml#L29-L30

Reply:
`/etc/ci/env/networking-info.yml` is not copied to the nested CRC VM AFAIU. I assume it's on the hypervisor though and I should be able to slurp it from there.

I will test and submit a follow up patch and ping you. I normally wouldn't hardcode such a thing but it's already hardcoded everywhere. I wanted to get the CIX fixed asap but will do it correctly now.